newm-server/src/main/kotlin/io/newm/server/features/idenfy/IdenfyKoinModule.kt

```diff
@@ -16,7 +18,7 @@ val idenfyKoinModule = module {
     single<Key>(IDENFY_KEY_QUALIFIER) {
         with(get<ApplicationEnvironment>()) {
             SecretKeySpec(
-                getConfigString("idenfy.signature.Key").toByteArray(),
+                runBlocking { getSecureConfigString("idenfy.signature.key").toByteArray() },
```

Comment:
sounds like in Koin v4, they are planning to implement coroutine support so we won't have to do runBlocking any longer.